networkx/utils/backends.py

```diff
@@ -418,8 +425,8 @@ def __new__(
             This overrides all the other preserve_*_attrs.
 
         mutates_input : bool or dict, default False
-            For bool, whether the functions mutates an input graph argument.
-            For dict of ``{arg_name: arg_pos}``, arguments that indicates whether an
+            For bool, whether the functions mutate an input graph argument.
```

Comment:
We're talking about a single function (the one that's wrapped by the decorator):
```suggestion
            For bool, whether the function mutates an input graph argument.
```